samples/apps/copilot-chat-app/importdocument/Config.cs

```diff
@@ -28,6 +28,11 @@ public sealed class Config
     public string ServiceUri { get; set; } = string.Empty;
 #pragma warning restore CA1056 // URI-like properties should not be strings
 
+    /// <summary>
+    /// Command path (relative URI) to which the file will be uploaded.
+    /// </summary>
+    public string CommandPath { get; set; } = string.Empty;
```

Comment:
I'm not sure this needs to be configurable. In our scenarios I don't think this would ever change since we're tied to the API contract of the CopilotChat WebAPI.

Reply:
Sure, will remove this configurable.